Sources/SwiftDocC/Infrastructure/Diagnostics/DiagnosticConsoleWriter.swift

```diff
@@ -36,8 +36,7 @@ public final class DiagnosticConsoleWriter: DiagnosticFormattingConsumer {
         self.init(stream, formattingOptions: options, baseURL: baseURL, highlight: highlight, fileManager: FileManager.default)
     }
     
-    @_spi(FileManagerProtocol)
-    public init(
+    package init(
```

Comment:
I couldn't because it's used in a public protocol and I couldn't use different access levels in the same protocol. 

We will eventually settle on a stable API for it and then we can change the SPI to public API.